### PR TITLE
remove webConsolePlugin from ToolchainClusterConfig (phase 2)

### PR DESCRIPTION
**Pull request by xcoulon**
Pendo has been decommisioned, we don't need this plugin anymore

see also:
- https://github.com/codeready-toolchain/api/pull/438
- https://github.com/codeready-toolchain/toolchain-common/pull/420 
- https://github.com/codeready-toolchain/toolchain-e2e/pull/1026 (standalone)

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: cmd/consoleplugin/main.go [hunk 1]
```diff
@@ -94,30 +85,7 @@ func main() {
...
-
-	gracefulShutdown(gracefulTimeout, pluginServer)
-}
-
-func startConsolePluginService(config membercfg.WebConsolePluginConfig) *consoleplugin.Server {
-	consolePluginServer := consoleplugin.NewConsolePluginServer(config, setupLog)
-	consolePluginServer.Start()
-
-	return consolePluginServer
+	gracefulShutdown(gracefulTimeout)
```
Comment (alexeykazakov) [inline cmd/consoleplugin/main.go:88]: Why do you keep it? We don't really care about graceful shutdown of the web console service, do we? Why not deleting the entire `cmd/consoleplugin/main.go` and then manually deleting already running web console plugin service/apps in clusters?

**Review by alexeykazakov — COMMENTED** [review]
1. Please create a separate PR for the CRD update. See https://github.com/codeready-toolchain/host-operator/pull/1068#discussion_r1693303743 for details.
2. We also need to delete https://github.com/codeready-toolchain/member-operator/tree/master/webconsole_source
3. And please clean up the make files (make generates plugin JS from the webconsole TS source a mentioned above)

File: cmd/consoleplugin/main.go (hunk 1 above)
Comment (xcoulon) [reply cmd/consoleplugin/main.go:88]: right, I missed that 🤦‍♂️

Comment (xcoulon) [reply cmd/consoleplugin/main.go]: removed in e99f22c

**Review by alexeykazakov — APPROVED** [review]

**Pull request merged**